spec/system/inactive_application_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.feature 'Process Stale applications', sidekiq: true, time: Time.zone.local(2023, 11, 11, 0, 0, 0) do
```

Comment:
It might be good to declare that the spec is testing with the continuous applications flag enabled.

> Process Stale applications with Continuous Applications